DATA/MISSIONS/mbases.ini

```diff
@@ -37289,13 +37289,13 @@ rumor = base_0_rank, mission_end, 1, 461597
 ; I have been with The Order for 6 years now, chosen for my piloting skills. There is no active recruiting going on right now, but if you can hold your own out there you can probably impress some of our operatives. If you can't, well, it was nice meeting you.
 rumor = base_0_rank, mission_end, 1, 461598
 ;res html order_toledo_rumor03
-; Nomads travel in packs, usually four ships at least. If you only see one, that usually means the nebula is hiding the others from your scanner. Don't get baited thinking you're seeing an easy target when you're about to become <i>their</i> easy target.
+; Nomads travel in packs, usually four ships at least. If you only see one, that usually means the nebula is hiding the others from your scanner. Don't get baited thinking you're seeing an easy target when you're about to become <I>their</I> easy target.

```

Comment:
The tags are not HTML tags. They require no <>. Check the FRC readme in "/resources/frc/" directory. Correct would be here to use `\i` `\I` I think.

Reply:
Considering that I used and corrected these in the titles of item infocards before, I should have known that. Whoops. Corrected now.